app/models/classification.rb

```diff
@@ -465,7 +471,9 @@ def validate_uniqueness_on_tag_name
   end
 
   def validate_format_of_name
-    errors.add("name", "must be lowercase alphanumeric characters and underscores without spaces") unless (name =~ /[^a-z0-9_:]/).nil?
+    unless (name =~ /[^a-z0-9_:]/).nil?
+      errors.add("name", "must be lowercase alphanumeric characters, colons and underscores without spaces")
+    end
```

Comment:
I was expecting you to just delete this validator and add a standard inclusion validator.

no requested change